script/src/verify.rs

```diff
@@ -853,18 +844,21 @@ impl<DL: CellDataProvider + HeaderProvider + ExtensionProvider + Send + Sync + C
 
         for (idx, (_hash, group)) in self.groups().enumerate().skip(current + 1) {
             let remain_cycles = limit_cycles.checked_sub(current_used).ok_or_else(|| {
-                ScriptError::Other(format!("expect invalid cycles {limit_cycles} {cycles}"))
-                    .source(group)
+                ScriptError::Other(format!(
+                    "here expect invalid cycles {limit_cycles} {cycles}"
```

Comment:
Do we need to append `{current_used}` to `format!`'s argument?

Reply:
better add it